src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/CreateIndexAttemptBuilder.java

```diff
@@ -49,7 +49,7 @@ public CreateIndexAttempt build() {
     // Validate required fields
     if (schemaObject == null || columnName == null || dataType == null || indexName == null) {
```

Comment:
pls follow standard and validate the class in the consturctor and use Objects.requireNotNull() with a message and what field is invalid.